### ISSUE TITLE
Notification of unread OSM messages

### ISSUE THREAD

**Issue by atarom** [systemed/potlatch3#38]
Hi, it seems that users who use potlach do not receive the notification of messages to be read in the application?
thanks in advance

**Comment by systemed** [comment]
That’s correct. Potlatch is an editor and not a client for interacting with every aspect of OSM.

**Comment by atarom** [comment]
ok, thanks for the quick response
I don't understand code, but if this could be implemented to let the user know he/she has messages to read it would be nice.
https://wiki.openstreetmap.org/wiki/API_v0.6#Details_of_the_logged-in_user
``
...
		<messages>
			<received count="1" unread="0"/>
			<sent count="0"/>
		</messages>
...
``

**Comment by systemed** [comment]
It's not something I have planned. osm.org already sends an email to users to alert them that they have a message, Potlatch doesn't need to duplicate that functionality. I wouldn't refuse a pull request to add it, of course.
